packages/ledger-amino/src/launchpadledger.ts

```diff
@@ -4,7 +4,8 @@ import { fromUtf8 } from "@cosmjs/encoding";
 import { assert } from "@cosmjs/utils";
 import Transport from "@ledgerhq/hw-transport";
 import CosmosApp, {
-  AppInfoResponse,
+  AddressAndPublicKeyResponse,
+  AppInfoResponse, ErrorResponse,
```

Comment:
Where is `ErrorResponse` defined? I did not find anything in https://github.com/Zondax/ledger-cosmos-js.

Is this the same we create as `export interface LedgerAppErrorResponse` a few lines below?